code/modules/hivemind/objects.dm

```diff
@@ -3,20 +3,20 @@
 
 //toxic shot, turret's ability use it
 /obj/item/projectile/goo
-	name = "acrid goo"
+	name = "acidic goo"
```

Comment:
Name it ``electrolyzed goo`` or something like that, because of the armor check change.

Reply:
Will do, sounds better anyways.

Reply:
Done so. Description also now mentions ozone as to hint even more at the fact that it deals energy damage